src/v/kafka/server/group.cc

```diff
@@ -1859,9 +1859,11 @@ group::handle_commit_tx(cluster::commit_group_tx_request r) {
     } else if (
```

Comment:
> Make it possible to modify the same consumer group concurrently
by multiple producers

Would it be accurate to add:

_Prior to this change a single lock serialized transaction. This commit introduces fine grained locking on a per producer_id basis_?

Reply:
changed